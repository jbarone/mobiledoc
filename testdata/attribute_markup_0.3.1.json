{
	"version": "0.3.1",
	"atoms": [],
	"cards": [],
	"markups": [
		["A", ["href", "http://google.com"]]
	],
	"sections": [
		[1, "P", [
			[0, [0], 1, "hello world"]]
		]
	]
}
