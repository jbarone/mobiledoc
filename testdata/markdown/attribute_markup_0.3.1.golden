[hello world](http://google.com)

